bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx75.html

```diff
@@ -0,0 +1,150 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_75" %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_75') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success up-to-date">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+
+      <div class="mzp-c-emphasis-box show-signed-out">
+        <img class="c-emphasis-box-hero" src="{{ static('img/firefox/whatsnew/whatsnew75/hero-sync.svg') }}" alt="">
+        <h2 class="c-emphasis-box-title">{{ _('Pick up Firefox where you left off') }}</h2>
+        <p>{{ _('Whether you’re using Firefox on your phone or computer, make sure all your important stuff — internet searches, passwords, open tabs — appears where you need it.') }}</p>
+
+        <p class="cta">
+          {# NOTE: The '/connect_another_device/' landing page is new so this is a kludge that duplicates the fxa_cta_link macro. If we do this again we should make a macro/helper for this button. #}
+          <a href="{{ settings.FXA_ENDPOINT }}connect_another_device/?form_type=button&entrypoint={{ entrypoint }}&utm_source={{ entrypoint }}&utm_medium=referral&utm_campaign={{ campaign }}&utm_content=whatsnew75-sync&context=fx_desktop_v3" data-action="{{ settings.FXA_ENDPOINT }}" data-mozillaonline-link="{{ settings.FXA_ENDPOINT_MOZILLAONLINE }}connect_another_device/??form_type=button&entrypoint={{ entrypoint }}&utm_source={{ entrypoint }}&utm_medium=referral&utm_campaign={{ campaign }}&utm_content=whatsnew75-sync&context=fx_desktop_v3" data-cta-type="whatsnew-75-sync" data-cta-position="primary" class="mzp-c-button mzp-t-product js-fxa-product-button qa-sync-button" rel="external">{{ _('Sync your phone') }}</a>
```

Comment:
Duplicate `?` on the mozillaonline link: `connect_another_device/??form_type=button`

```suggestion
          <a href="{{ settings.FXA_ENDPOINT }}connect_another_device/?form_type=button&entrypoint={{ entrypoint }}&utm_source={{ entrypoint }}&utm_medium=referral&utm_campaign={{ campaign }}&utm_content=whatsnew75-sync&context=fx_desktop_v3" data-action="{{ settings.FXA_ENDPOINT }}" data-mozillaonline-link="{{ settings.FXA_ENDPOINT_MOZILLAONLINE }}connect_another_device/?form_type=button&entrypoint={{ entrypoint }}&utm_source={{ entrypoint }}&utm_medium=referral&utm_campaign={{ campaign }}&utm_content=whatsnew75-sync&context=fx_desktop_v3" data-cta-type="whatsnew-75-sync" data-cta-position="primary" class="mzp-c-button mzp-t-product js-fxa-product-button qa-sync-button" rel="external">{{ _('Sync your phone') }}</a>
```